README.md

```diff
@@ -4,10 +4,10 @@ This action build Android project, export .apk file as GitHub artifact, with opt
 
 Tested with Ionic, React Native and native android projects.
 
-## New in 1.2.0
+## New in 1.3.0
```

Comment:
@EmilianoMorghen rather than deleting the 1.2 version section, add the 1.3 details above it

Reply:
Done

Reply:
Done